doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_reg.rst

```diff
@@ -0,0 +1,20 @@
+.. _bt_mesh_light_ctrl_reg_readme:
+
+Light Lightness Control regulator
+#################################
+
+This module is an abstract PI regulator. Regulator implementations should declare a
+:c:struct:`bt_mesh_light_ctrl_reg` struct and supply implementations for ``init``, ``start`` and ``stop``.
+
+For an example of a regulator implementation, see :ref:`bt_mesh_light_ctrl_reg_readme`,
```

Comment:
```suggestion
For an example of a regulator implementation, see :ref:`bt_mesh_light_ctrl_reg_readme`.
```
Also, it refers to itself, is it supposed to? And does it make sense?

Reply:
Oh yeah that's an error, was supposed to refer to reg_spec. Will fix